contrib/node/src/python/pants/contrib/node/tasks/javascript_style.py

```diff
@@ -36,15 +36,15 @@ def register_options(cls, register):
     register('--skip', type=bool, fingerprint=True, help='Skip javascriptstyle.')
     register('--fail-slow', type=bool,
              help='Check all targets and present the full list of errors.')
-    register('--javascriptstyle-dir', advanced=True, fingerprint=True,
```

Comment:
I'm ok with removing the functionality behind this option because it was added during the 1.4.0 cycle. But I think you should still leave the option as deprecated, so that anyone who might have used it will get some hint of where to look for the new solution.

See https://github.com/pantsbuild/pants/blob/fb0ebf720b73a65f10534fb70ceab38df77aaad3/src/python/pants/backend/codegen/thrift/java/thrift_defaults.py#L26-L29 for an example of how to deprecate an option. You should target `1.6.0.dev0` for the removal.